### ISSUE TITLE
playbook-control-room-desktops.yml has grown too large in scope

### ISSUE THREAD

**Issue by lerwys** [lnls-sirius/lnls-ansible#3]
**Problem:** playbooks should have fixed hosts targets. playbook-control-room-desktops.yml, on the other hand, keeps growing in the number of hosts groups it acts, making it not only for "control-room". I think we should keep playbooks at a minimum set of hosts and a generic one to be used in general. What do you think @xresende ?

**Solution:** Sanitize playbooks to act on specific host groups and create a generic one with "hosts: all" to be used on demand by setting the "HOSTS" make variable

**Comment by xresende** [comment]
agreed, @lerwys. i will start sanitizing the playbooks and start updating our makefile targets.

**Comment by lerwys** [comment]
Nice! Thanks
